app/src/main/java/xyz/whereuat/whereuat/Contact.java

```diff
@@ -2,45 +2,29 @@
 
 import android.content.ContentValues;
 import android.content.Context;
-import android.util.Log;
 
-import xyz.whereuat.whereuat.db.DbObject;
-import xyz.whereuat.whereuat.db.DbTask;
 import xyz.whereuat.whereuat.db.command.InsertCommand;
+import xyz.whereuat.whereuat.db.command.QueryCommand;
 import xyz.whereuat.whereuat.db.entry.ContactEntry;
 
 /**
  * Created by kangp3 on 4/10/16.
  */
-public class Contact implements DbObject {
-    public final String name;
-    public final String phone;
-    public final boolean autoshare;
-
+public class Contact {
```

Comment:
I'm opposed to this just because it could also be used for utility methods (see the `KeyLocation` class)